espnet2/tts/gradtts/denoiser.py

```diff
@@ -0,0 +1,324 @@
+import math
+
+import torch
+from einops import rearrange
+
+from espnet2.tts.gradtts.base import BaseModule
+
+
+class Mish(BaseModule):
```

Comment:
```suggestion
class Mish(torch.nn.Module):
```
As suggested, use torch.nn.Module